# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: __dir__.ini
# Commit: 680c502179c6
# Full Hash: 680c502179c6a98e8e6960fbab016d3aea44ff4c
# Author: Andreas Farre <farre@mozilla.com>
# Date: 2023-05-10 21:37:01
# Regressor Bug: 1823877
# File Overlap Count: 1
# Description:
#   Bug 1823877 - Part 1: Filter opaque results from fetch() in the parent for ORB. r=sefeng,smaug,necko-reviewers,edenchuang,valentin
#   
#   We make sure to not send any data to the content process in case of
#   fetching an opaque resource. This is way to remain more web
#   compatible, but is also in conflict with the ORB specification.
# ==============================================================================

diff -r 8295c1c7bfa0 -r 680c502179c6 testing/web-platform/mozilla/meta/fetch/orb/__dir__.ini
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/testing/web-platform/mozilla/meta/fetch/orb/__dir__.ini	Wed May 10 14:35:52 2023 +0000
@@ -0,0 +1,3 @@
+prefs: [browser.opaqueResponseBlocking: true,
+        browser.opaqueResponseBlocking.filterFetchResponse:2]
+        
\ No newline at end of file